utils.js

```diff
@@ -95,14 +95,7 @@ function readRegistry (callback) {
 
                 if (success) {
                     try {
-                        let max_size = SettingsSchema.get_int(Prefs.Fields.HISTORY_SIZE);
```

Comment:
Hmm, why did you remove this?

Reply:
To avoid deleting the favourites in the init.

Reply:
I put again this code but without deleting favorites. See 4149e96